wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -261,6 +261,18 @@ def _get_return_node_variables(
                     returns[variable_name].append(sub_node.value)
         return returns
 
+    def _get_return_nodes(
```

Comment:
These two pieces of code are almost identical:

```py
def _get_return_node_variables(
        self,
        node: List[ast.AST],
    ) -> Dict[str, List[ast.Name]]:
        returns: Dict[str, List[ast.Name]] = defaultdict(list)
        for sub_node in node:
            if isinstance(sub_node, ast.Return):
                if isinstance(sub_node.value, ast.Name):
                    variable_name = sub_node.value.id
                    returns[variable_name].append(sub_node.value)
        return returns

    def _get_return_nodes(
        self,
        node: List[ast.AST],
    ) -> Dict[str, ast.Return]:
        return_sub_nodes: Dict[str, ast.Return] = defaultdict()
        for sub_node in node:
            if isinstance(sub_node, ast.Return):
                if isinstance(sub_node.value, ast.Name):
                    variable_name = sub_node.value.id
                    return_sub_nodes[variable_name] = sub_node
        return return_sub_nodes
```

Let's improve it!

Reply:
We can return tuples of `(return_node, variable_names)`. Any other solution is also fine.

Reply:
I tried this way. It works fine. The problem is that annotation like this:
```
def _get_return_node_variables(
    self,
    node: List[ast.AST],
) -> Tuple[Dict[str,  List[ast.Name]], Dict[str, ast.Return]]:
```
leads to violations:
```
./wemake_python_styleguide/visitors/ast/keywords.py:255:10: TAE002 too complex annotation (4 > 3)
./wemake_python_styleguide/visitors/ast/keywords.py:255:10: Z221 Found line with high Jones Complexity: 17
```

So, don't want to rewrite the whole visitor. I'll think more about this problem and will try to solve it.

Reply:
Just create a type alias somewhere at the top: `NamesAndReturns = Tuple[Dict[str,  List[ast.Name]], Dict[str, ast.Return]]`

Reply:
Oh, saw this many times and didn't guess to use..